backend/src/db/migrations/20240429172301_access_approval_requests.ts

```diff
@@ -31,21 +37,39 @@ export async function up(knex: Knex): Promise<void> {
   if (!(await knex.schema.hasTable(TableName.AccessApprovalRequestReviewer))) {
     await knex.schema.createTable(TableName.AccessApprovalRequestReviewer, (t) => {
       t.uuid("id", { primaryKey: true }).defaultTo(knex.fn.uuid());
-      t.uuid("member").notNullable();
-      t.foreign("member").references("id").inTable(TableName.ProjectMembership).onDelete("CASCADE");
+
+      t.uuid("memberUserId").notNullable();
+      t.foreign("memberUserId").references("id").inTable(TableName.Users).onDelete("CASCADE");
+
       t.string("status").notNullable();
       t.uuid("requestId").notNullable();
       t.foreign("requestId").references("id").inTable(TableName.AccessApprovalRequest).onDelete("CASCADE");
       t.timestamps(true, true, true);
     });
   }
+
+  if (!(await knex.schema.hasColumn(TableName.ProjectUserAdditionalPrivilege, "groupMembershipId"))) {
```

Comment:
Kindly create a separate table for `GroupAdditionalPrivilege` as this aligns like `IdentityUserAdditionalPrivilege`